services/crud_helper.py

```diff
@@ -35,10 +36,22 @@ def model_adder(session, table, model, user=None, **kwargs):
         md["created_by_id"] = user["sub"]
         md["created_by_name"] = user["name"]
 
+    notes = None
+    if issubclass(table, NotesMixin):
```

Comment:
The `NotesMixin` is to add the `notes` attribute to a table. Should this be `if issubclass(table, Notes)`?